src/frontend/js/components/DeletionModal.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { Button, Typography, Modal } from 'material-ui'
+import { withStyles } from 'material-ui/styles'
+
+function getModalStyle () {
+  const top = 50
+  const left = 50
+
+  return {
+    top: `${top}%`,
+    left: `${left}%`,
+    transform: `translate(-${top}%, -${left}%)`
+  }
+}
+
+const styles = theme => ({
+  paper: {
+    position: 'absolute',
+    backgroundColor: 'white',
+    padding: 10,
+    border: '3px solid purple'
+
+  },
+  form: {
+    float: 'right',
+    clear: 'none'
+  },
+  deleteImage: {
+    height: 100,
+    width: 100,
+    margin: 'auto',
+    display: 'block'
+  },
+  button: {
+    color: 'purple'
+  }
+})
+
+class DeletionModal extends React.Component {
+  constructor () {
+    super()
+    this.state = {
+      modalIsOpen: true
+    }
+    this.closeModal = this.closeModal.bind(this)
```

Comment:
In the `render` function you call `closeModal` from within a fat arrow function so there is no need to bind `this` here